bedrock/mozorg/helpers/misc.py

```diff
@@ -462,8 +462,10 @@ def absolute_url(url):
 
     if url.startswith('//'):
         prefix = 'https:'
-    else:
+    elif url.startswith('/'):
```

Comment:
I actually don't see a test for the new "else" branch of this code. Should have a unit test in there to ensure what you expect happens when the function is passed something that doesn't start with a slash.